src/vhive.rs

```diff
@@ -0,0 +1,146 @@
+//! This module uses `kubectl` to get a containerd id, uses `journalctl` to match that id to a
+//! firecracker vm id and finally searches the open file-descriptors of relevant processes for one
+//! belonging to that vm id.
+//!
+//! The user may supply the same `container_id` as for the kubnetes module, the container always
+//! defaults to `user-container` because that is where the firecracker vm resides in vhive.
+//!
+//! Requires:
+//! - journald size longer than container lifetime
+//! - vhive with "debug log containerid->vmid" patch
```

Comment:
I wonder if we could ask Dimitri to get an API endpoint for that... But works for now I guess.

Reply:
the chances are good that i will add api endpoints to vhive. I could add this cid->vmid there.